src/Control/Monad/Aff/Console.purs

```diff
@@ -14,4 +14,4 @@ log = liftEff <<< C.log
 -- | Prints any `Show`-able value to the console. This basically saves you
 -- | from writing `liftEff $ print x` everywhere.
 print :: forall e a. (Show a) => a -> Aff (console :: C.CONSOLE | e) Unit
-print = liftEff <<< C.print
+print = liftEff <<< C.logShow
```

Comment:
I think it would be better to be consistent with naming here and change print to logShow for aff aswell. Your call.